Source/SteveSloth/Private/MyEnemy.cpp

```diff
@@ -0,0 +1,20 @@
+// Fill out your copyright notice in the Description page of Project Settings.
```

Comment:
First please add this to the top and enter your information accordingly on both the cpp and the h files

/****************************************************************************************
* Copyright: SteveSloth
 * Name: Aidan
 * Script: MyEnemy.cpp
 * Date: April 23. 2024
 * Description: 
 * TODO:
 * Known Bugs:
 
 * TODO:
 * Known Bugs:
 ****************************************************************************************/